src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+.first {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
```

Comment:
Avoid using tag names for styling, as per the checklist. It's better to use class names instead. This will make your CSS more scalable and easier to maintain.